### PR TITLE
Integrate Search with Add Product

### PR DESCRIPTION
**Pull request by hibatawfeeq**
@Alikpld and I integrated the search with add user stories and should be reflected in this branch. We are still in process to finalize it.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,22 +19,23 @@ express()
...
   .get('/', jsonParser, async (req, res) => {
     try {
-      const client = await pool.connect();
-      const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
-      const results = { 'results': (result) ? result.rows : null};
-      res.render('pages/index', results );
-      client.release();
+        const client = await pool.connect();
+        const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
+        const searchresult = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')`);
```
Comment (nmdoran) [inline index.js]: Add additional where condition
WHERE user_ID IN ('0', '1')

File: index.js [hunk 2]
```diff
@@ -19,22 +19,23 @@ express()
   .set('view engine', 'ejs')
   .get('/', jsonParser, async (req, res) => {
     try {
-      const client = await pool.connect();
-      const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
-      const results = { 'results': (result) ? result.rows : null};
-      res.render('pages/index', results );
-      client.release();
+        const client = await pool.connect();
+        const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
```
Comment (nmdoran) [inline index.js]: Do we need to limit in user_ID here?

File: index.js [hunk 3]
```diff
@@ -19,22 +19,23 @@ express()
...
       console.error(err);
       res.send("Error " + err);
     }
 })
-  .get('/db', jsonParser, async (req, res) => {
+  .get('/addProduct', jsonParser, async (req, res) => {
     try {
       const client = await pool.connect();
-      const result = await client.query(`SELECT * FROM products where prod_name='${req.query.searchParam}'`);
+      const result = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')`);
```
Comment (nmdoran) [inline index.js:47]: Add additional where condition
WHERE user_ID IN ('0', '1')

**Review by nmdoran — COMMENTED** [review]
Suggestions on how to modify queries based on the changes to the database.

File: index.js (hunk 1 above)
Comment (nmdoran) [reply index.js]: Also, change the natural join to a left join on prod_ID

File: index.js (hunk 2 above)
Comment (nmdoran) [reply index.js]: Also the natural join will likely cause an issue here.  Change to a left join.

File: index.js (hunk 1 above)
Comment (hibatawfeeq) [reply index.js]: Perfect thanks Nicole just fixed it!

**Pull request merged**